src/integTest/groovy/com/bmuschko/gradle/docker/ToolingApiIntegrationTest.groovy

```diff
@@ -60,4 +60,8 @@ repositories {
     protected String createUniqueImageId() {
         "gradle/${UUID.randomUUID().toString().replaceAll('-', '')}"
     }
+
+    protected String createUniqueContainerName() {
```

Comment:
Instead of adding a new method I think we should simply rename the existing method `createUniqueImageId ` to something else. Probably just `createUniqueId`.